README.md

```diff
@@ -170,6 +170,16 @@ If you wish to make larger changes we generally recommend [forking](https://help
 
 ### Build Hets using Stack
 * [Install Stack](https://docs.haskellstack.org/en/stable/install_and_upgrade) (use the generic Linux option if you are on Ubuntu).
+* Install build- and GUI-dependencies
+  * Ubuntu:
+    ```
+    sudo apt-get install libglib2.0-dev libcairo2-dev libpango1.0-dev libgtk2.0-dev libglade2-dev libncurses-dev
+    ```
```

Comment:
Do we need/want to have the [quoted improvements of `terminfo`](https://github.com/spechub/Hets/pull/1708#discussion_r121206506) for the interactive interface? If we don't, we can remove the dependency of `libncurses-dev`.

If I remember correctly, the interactive interface may draw multiple lines, so removing `terminfo` could be a step back.

From the perspective of a user, I would like to have the "forwards delete" feature. So it's a plus for `terminfo`.